docs/getting-started/tutorial-sample-filter.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[tutorial-sample-filter]]
+=== Filtering the data
+
+Many visualizations in the *Global Flight* dashboard are interactive. They offer 
+mouse clicks and other controls for you to filter the view of the data. Filters 
+are global&mdash;a change in one visualization simultaneously changes the data 
+in every visualization on the dashboard.
+
+. In the *Controls* visualization, set an *Origin City* and a *Destination City*.
+. Click *Apply changes*.
++
+The `OriginCityName` and the `DestCityName` fields are filtered to match 
+the data you specified. 
++
+For example, this dashboard shows the data for flights from London to Newark
+and Pittsburgh. 
++
+[role="screenshot"]
+image::images/tutorial-sample-filter.png[]
++
+. Remove the filters by hovering your pointer over their labels in the filter bar
+and clicking the trash can icon. 
```

Comment:
Maybe, "The applied filters are shown in the filter bar. To remove a filter, click it's trash icon."?  (Not sure it's really necessary to say "trash can" at this point?)

Reply:
Actually, there may be a better way using the input controls here. Rather than hitting the filters and deleting them one by one. You could guide users to hit `Clear form` and `Apply changes` this will remove all filters the input controls added.